new-version.sh

```diff
@@ -15,68 +15,74 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-# Usage: $ ./new-version.sh <VERSION>
+# Usage: $ ./new-version.sh <NEW_VERSION> <NEXT_VERSION>
 
 set -e
 
-# PYTHON
+usage() {
+  echo "usage: ./$(basename -- ${0}) <NEW_VERSION> <NEXT_VERSION>"
+  exit 1
+}
+
+readonly SEMVER_REGEX="^[0-9]+(\.[0-9]+){2}(-rc\.[0-9]+)?(-SNAPSHOT)?$" # X.Y.Z
```

Comment:
would we use both RC.X _and_ -SNAPSHOT? I'd imagine one or the other.

Reply:
Woah! Nice catch 👍